drivers/i2c/Kconfig.omap

```
@@ -0,0 +1,18 @@
+# Copyright (C) 2024 BeagleBoard.org Foundation
+# Copyright (C) 2024 Dhruv Menon <dhruvmenon1104@gmail.com>
+
+# SPDX-License-Identifier: Apache-2.0
+
+config I2C_OMAP
+	bool "TI OMAP I2C Driver"
+	default y
+	depends on DT_HAS_TI_OMAP_I2C_ENABLED
+	help
+		Enable the I2C driver for TI OMAP SoCs.
```

Help text should be indented with a tab and 2 spaces.
```suggestion
	  Enable the I2C driver for TI OMAP SoCs.
```

---

Updated it.